source/shared/cpp/ObjectModel/SharedAdaptiveCard.cpp

```diff
@@ -139,6 +139,12 @@ std::shared_ptr<ParseResult> AdaptiveCard::Deserialize(
     auto result = std::make_shared<AdaptiveCard>(version, fallbackText, backgroundImage, style, speak, language, body, actions);
     result->SetLanguage(language);
 
+    // Parse selectAction if present (selectAction is optional so isRequired defaults to false)
+    auto selectAction = ParseUtil::GetSelectAction(elementParserRegistration, actionParserRegistration, json, AdaptiveCardSchemaKey::SelectAction, false);
+    if (selectAction != nullptr) {
+        result->SetSelectAction(selectAction);
+    }
```

Comment:
[nit] formatting

Reply:
can you elaborate?

Reply:
Hey Tuan, 
If you are commenting using git hub web interface, please install CodeFlow plugin and use CodeFlow. 

---
In reply to: [182211561](https://github.com/Microsoft/AdaptiveCards/pull/1380#discussion_r182211561) [](ancestors = 182211561)